### PR TITLE
bug/match metamask network on wallet unlock

### PR DESCRIPTION
**Pull request by SteveMieskoski**
### Bug
- [x] Updated CHANGELOG.md
- [X] Add PR label

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 ### Release v5.1.2
```
Comment (gamalielhere) [inline CHANGELOG.md]: should probably remove this

File: src/networks/types/GOERLI.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { GOERLI } from '../tlds';
 import goerli from '@/assets/images/icons/network.svg';
 
 export default {
-  name: 'GöETH',
+  name: 'GOERLI',
```
Comment (kvhnuke) [inline src/networks/types/GOERLI.js]: why did you change this to `GOERLI` ?

Comment (SteveMieskoski) [reply src/networks/types/GOERLI.js]: There was an issue with matching the name as it is different than the type file name.  I don't remember where it was occurring, so I just reverted it.
